.github/workflows/pytest.yml

```diff
@@ -10,6 +10,7 @@ on:
     branches:
       - main
       - releasebranch_*
+      - "*"
```

Comment:
This means PRs against any branch, no? Why not making this shorter, e.g., by leaving out the whole branches part.

Reply:
Yah, I could. Since it wasn't the purpose of the PR, I only needed it to be able to run the CI on my fork. My main branch is still configured for another goal, for code coverage, so I needed it to run on another branch (I'm at main8 currently). Having it that way I would be able to just remove the line with a rebase (dropping the commit). I wanted to finish the feedback before changing it final. If you think we could allow running on all (it won't automatically run on older branches, as it isn't the content of the ref), I would change to only `pull_request:`.

I'd be happy if it was, as I'm doing these changes for every PR on my side to test them.

Reply:
I'm removing the branch filtering

Reply:
The workflow file needs to be on the branch to get triggered, no? So, specifying them here is maybe _a_ practice which does not have any useful application in our repository.

Reply:
> The workflow file needs to be on the branch to get triggered, no? So, specifying them here is maybe _a_ practice which does not have any useful application in our repository.

For pull_request, it seems like it is the case. But for other triggers, it looks like it is the contents of the default branch (main) that counts, like pull_request_trigger. For workflow_dispatch, to manually run a workflow, I already saw a branch selector to choose what version of the workflow to dispatch.